uber_analytics/templates/graphs/index.html

```diff
@@ -2,13 +2,13 @@
 {% block title %}Attendance Graphs{% endblock %}
 {% block head_additional %}
     <!-- <script type="text/javascript" src="https://www.google.com/jsapi"></script> dont think we need this? -->
```

Comment:
Ah right - the analytics plugin mounts its own static files under ``/uber/analytics/static``. That used to be the intended way to do this, and it's still fine to do it that way.  However, we do now have a ``static_overrides()`` utility method that can be called, so that's a change that could be made in the future if we wanted.